draft-pauly-oblivious-doh.md

```diff
@@ -468,18 +468,23 @@ back to the client as the HTTP response to the client's original HTTP request.
 DISCLAIMER: this is a work in progress draft and has not yet seen significant security analysis.
 
 Oblivious DoH aims to keep knowledge of the true query origin and its contents known to only
-clients. In particular, it assumes a Dolev-Yao style attacker which can observe all client queries,
-including those forwarded by oblivious proxies, and does not collude with target resolvers. (Indeed,
-if a target colludes with the network attacker, then said attacker can learn the true query origin
-and its contents.) Moreover, Oblivious DoH assumes that Proxies and Targets are not malicious and
-they do not collude. Malicious Targets are out of scope because DNS recursive resolvers behind
-targets may reply with legitimate or bogus answers in the absence of DNSSEC, so Target malfeasance
-does not introduce a unique threat. Moreover, malicious Proxies are out of scope because clients must
-trust them to access Targets. Malicious Proxies could block or otherwise tamper with Oblivious DoH
-queries, though this is not different from a malicious network which blocks client access to a
-DoH server.
-
-Oblivious DoH aims to achieve the following confidentiality goals in the presence of this threat model:
+clients. In particular, it assumes an extended Dolev-Yao style attacker which can observe all client
+queries, including those forwarded by oblivious proxies, and does not collude with target resolvers.
```

Comment:
I was thinking it could collude with target resolvers, I was assuming it just couldn't collude with both the proxy and the target at the same time.

Reply:
Yeah, I should have updated this. The text below suggests that it can compromise the target or proxy, anyway.

Reply:
@jhoyla -- does the new text look good to you? I think it clarifies things.